<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Ilan Ben Bassat (Tel Aviv)</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-item"><a href="../index.html">Seminar&nbsp;home</a></div>
<div class="menu-item"><a href="../past.html">Past&nbsp;Talks</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Ilan Ben Bassat (Tel Aviv)</h1>
</div>
<p>Sep 19, 2016. 3-4PM, Cory 400.</p>
<h2>Title and Abstract</h2>
<p><b>Hashing and Sampling Techniques for Genome Assembly</b> <br />
New sequencing technologies generate larger amounts of data at decreasing costs. De novo sequence assembly is the problem of combining these reads back to the original genome sequence, without relying on a reference genome. This presents algorithmic and computational challenges, especially for long and repetitive genome sequences. </p>
<p>One of the main paradigms of gnome assembly is the Overlap-Layout-Consensus approach. Assemblers of that kind process the raw sequence data by traversing a graph that captures the overlaps between the reads (overlap graph). Although an extensive research in this area has been conducted in the past two decades, there is a constant need to optimize the performance of existing assemblers.</p>
<p>In the first part of the talk we will describe a simple approach for constructing a lightweight version of an overlap graph, based on special hash functions and Bloom filters. </p>
<p>In the second part we will discuss how to use sampling in order to improve the assembly of Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), a fascinating bacterial immune mechanism against phages. Our algorithm involves a series of partial constructions of the overlap graph, and it avoids part of the difficulties that other assemblers face when dealing with genomic repeats.</p>
<h2>Bio</h2>
<p>Ilan Ben-Bassat is a PhD student at the School of Computer Science of Tel-Aviv University, under the supervision of Prof. Benny Chor. His research focuses in designing computational methods for deep-sequencing data analysis. Ilan holds MSc degree and BSc degree in Computer science, both of which from Tel-Aviv University. He is currently a visiting researcher student in California Institute of Technology (Caltech), hosted by Prof. Jehoshua (Shuki) Bruck. His current project explores how to better model human creative thinking</p>
<div id="footer">
<div id="footer-text">
Page generated 2016-09-16 10:03:35 PDT, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
